zerver/views/realm.py

```diff
@@ -84,7 +83,6 @@ def update_realm(
             Realm.EMAIL_ADDRESS_VISIBILITY_TYPES), default=None),
         default_twenty_four_hour_time: Optional[bool]=REQ(validator=check_bool, default=None),
         video_chat_provider: Optional[int]=REQ(validator=check_int, default=None),
-        google_hangouts_domain: Optional[str]=REQ(validator=check_string, default=None),
```

Comment:
Can you document this API change via API_FEATURE_LEVEL and updating `templates/zerver/api/changelog.md`?

We don't have documentation for this endpoint, so that's all we need to do to document the change.